1.8.9/src/main/resources/assets/axolotlclient/lang/en_us.json

```
@@ -13,5 +13,6 @@
 	"runGCOnUnfocus": "Run GC on Unfocus",
 	"scrollbarColor": "Scrollbar Color",
 	"unfocusedFPS": "Unfocused FPS",
-	"unfocusedVolumeMultiplier": "Unfocused Volume Multiplier"
+	"unfocusedVolumeMultiplier": "Unfocused Volume Multiplier",
+	"alignToBottom": "Align to Bottom Screen edge"
```

```suggestion
	"alignToBottom": "Align to Bottom Screen Edge"
```
would be more correct afaik